### PR TITLE
fix(documentation): add map navigation

### PR DESCRIPTION
**Pull request by goncalosard**
## Summary

Implemented the map navigation page, and set the previews for the page.

## How did you test this change?

Tested locally

### CODE REVIEW

**Review by danielleroux — CHANGES_REQUESTED** [review]
Did you forgot to add the components from the different test-apps, looks like you only added the auto-generated markdowns but not the source files?

- Missing /Users/daniel/dev/oss/ix/packages/react-test-app/src/preview-examples/map-navigation-overlay.tsx
- Missing /Users/daniel/dev/oss/ix/packages/react-test-app/src/preview-examples/map-navigation.tsx
- Missing /Users/daniel/dev/oss/ix/packages/angular-test-app/src/preview-examples/map-navigation-overlay.ts
- Missing /Users/daniel/dev/oss/ix/packages/angular-test-app/src/preview-examples/map-navigation.ts
- Missing /Users/daniel/dev/oss/ix/packages/html-test-app/src/preview-examples/map-navigation.html
- Missing /Users/daniel/dev/oss/ix/packages/html-test-app/src/preview-examples/map-navigation-overlay.html

**Review by danielleroux — COMMENTED** [review]
@nuke-ellington Can you please have a look if the map-navigation have a problem with the overlay? Looks a little bit broken the map-navigation

image

**Review by danielleroux — APPROVED** [review]

**Comment by nuke-ellington** [comment]
@danielleroux I created an issue for the sidebar issues
@goncalosard the main content area in the examples should not be populated with the text "sidebar content"

**Review by danielleroux — CHANGES_REQUESTED** [review]
After building the branch locally i got changes in the auto-generated files. This should not happen please update files

**Comment by danielleroux** [comment]
https://github.com/siemens/ix/issues/92

File: packages/documentation/docs/controls/navigation/map-navigation.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  </TabItem>
+  <TabItem value="angular">
+    <AngularOverlayComponent />
+  </TabItem>
+</Preview>
+
+## Change language of legal links
+
+Supported language codes are `'global/en' | 'global/es' | 'de/de' | 'cn/zh'`
+
```
Comment (danielleroux) [inline packages/documentation/docs/controls/navigation/map-navigation.md:52]: Not existing anymore

File: packages/html-test-app/src/preview-examples/map-navigation-overlay.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
+<!--
```
Comment (danielleroux) [inline packages/html-test-app/src/preview-examples/map-navigation-overlay.html:1]: Is this working not at my side

**Review by danielleroux — CHANGES_REQUESTED** [review]

**Review by danielleroux — APPROVED** [review]

**Pull request merged**